src/configuration.ts

```diff
@@ -1608,7 +1601,7 @@ export async function selectTarget(): Promise<void> {
         if (configureAfterCommand) {
             // The set of build targets remains the same even if the current target has changed
             logger.message("Automatically reconfiguring the project after a build target change.");
-            await make.configure(make.TriggeredBy.configureAfterTargetChange, false);
+            await make.cleanConfigure(make.TriggeredBy.configureAfterTargetChange, false);
```

Comment:
I assume this modification is due to the same reason as here: https://github.com/microsoft/vscode-makefile-tools/pull/491/files#r1269821024?

Reply:
yes